### ISSUE TITLE
Hello World package name issue

### ISSUE THREAD

**Issue by benmccann** [micronaut-projects/micronaut-core#3102]
Page: https://docs.micronaut.io/2.0.0.M2/guide/index.html#creatingServer
Source: https://github.com/micronaut-projects/micronaut-core/blob/master/src/main/docs/guide/quickStart/creatingServer.adoc

The docs say to run `mn create-app hello-world`. This creates a new project with a single Java file available at `src/main/java/micronaut/hello/world/Application.java`

The docs then say "place the previous file in `src/main/java/hello/world`". This seems wrong to me because the previously created `Application.java` has `micronaut` in the path.

Is this a mistake that `micronaut` was left out of the second path? If not, it may be clearer to use paths that are more differentiated and meaningful such as `server` and `controller`

**Comment by graemerocher** [comment]
Seems like a mistake, would you like to send a PR?

**Comment by jameskleeh** [comment]
> This creates a new project with a single Java file available at src/main/java/micronaut/hello/world/Application.java 

That isn't true

```
sample-projects $ mn -V
| Micronaut Version: 1.3.4
| JVM Version: 1.8.0_212
sample-projects $ mn create-app hello-world
| Generating Java project...
| Application created at /Users/jameskleeh/IdeaProjects/sample-projects/hello-world
sample-projects $ ll hello-world/src/main/
java/      resources/ 
sample-projects $ ll hello-world/src/main/java/hello/world/Application.java 
-rw-r--r--  1 jameskleeh  staff  184 Apr 29 01:21 hello-world/src/main/java/hello/world/Application.java
```

**Comment by benmccann** [comment]
I think I figured out what happened here. I created my app with `mn create-app micronaut-hello-world` because I have a ton of different projects and wanted to name it something where I'd be able to remember what it was. I had no expectation that changing the project name would change the package name. It might make more sense to just always use `io.micronaut.example` as the package name? Or maybe to document that the package name is based off the project name?
